src/main.js

```diff
@@ -15,26 +15,30 @@ const { zipNodeJs } = require('./zip_node')
 const zipFunctions = async function (
   srcFolder,
   destFolder,
-  { parallelLimit = DEFAULT_PARALLEL_LIMIT, skipGo, zipGo } = {},
+  { parallelLimit = DEFAULT_PARALLEL_LIMIT, skipGo, zipGo, nodeResolvePaths = [] } = {},
```

Comment:
Note: we need to pass `{buildDir}/.netlify/plugins/node_modules` from both:
  - `@netlify/build` (core functions bundling)
  - `@netlify/functions-util`

Reply:
Also to https://github.com/netlify/js-client/blob/446c04f078c6044863c258aad5dab6e0d1970077/src/deploy/hash_fns.js#L15